<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskExtractionClass</class>
 <widget class="QWidget" name="MaskExtractionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaskExtraction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLineEdit" name="lineEditInputRaster">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输入栅格：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditInputMask">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输入栅格或矢量（掩膜数据）：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditOutputRaster">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出栅格：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="inputRasterButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/openFile.png</normaloff>:/login/res/openFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="inputMaskButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/openFile.png</normaloff>:/login/res/openFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="outputRasterButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="login.qrc"/>
 </resources>
 <connections/>
</ui>
